tests/test_sft_trainer.py

```diff
@@ -168,108 +168,108 @@ def test_parse_arguments_peft_method(job_config):
 ############################# Prompt Tuning Tests #############################
 
 
-def test_run_causallm_pt_and_inference():
```

Comment:
Instead of commenting out these tests, lets add a pytest skip instead like `@pytest.mark.skip(reason="currently crashes before validation is done")` as seen [here](https://github.com/foundation-model-stack/fms-hf-tuning/blob/844b61f51005a012dba2620974a9fc218ff99e78/tests/test_sft_trainer.py#L607) and then provide the reason and error

Reply:
Thank you for reviewing the PR Anh. I have added this in all 3 test cases.